test/esinstall/package-node-fetch/package-node-fetch.test.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+const {install} = require('../../../esinstall/lib');
+
+describe('package node-fetch', () => {
+  it('allows importing node-fetch', async () => {
+    const cwd = __dirname;
+    const dest = path.join(cwd, 'test-node-fetch');
+    const spec = 'node-fetch';
+
+    const {
+      importMap: {imports}
+    } = await install([spec], {
+      cwd,
+      dest
+    });
+
+    expect(imports[spec]).toBeTruthy();
```

Comment:
I was debating if here it’d be good to test that the polyfill kicked in. On the one hand, I don’t want this to be too-scoped to the polyfill unless it changes. But we want to ensure it didn’t leave `node-fetch` untouched.

WDYT about checking for this string, or something close to?

```js
global.fetch.bind(global);
```

Reply:
I was thinking about `import()`ing it dynamically, would that work?

Reply:
If you imported it, you may have to deal with Windows differences (line breaks, or sometimes even Windows-specific code; I see `posix` in the generated code which concerns me).

You could also simply verify that `src !== dest` too—just verify that it changed, rather than dealing with polyfill changes.

Overall though I could be wrong about this, but I don’t think the polyfill changes too much over time nowadays, esp. since the API must stay stable, if you could just find an unopinionated line to check for that must match the Node spec.

Reply:
Added this.